config/jobs/kubernetes/sig-node/dynamic-resource-allocation.jinja

```diff
@@ -0,0 +1,115 @@
+{%- if beginning %}
+{%- if kind == "ci" %}
+periodics:
+{%- else %}
+presubmits:
+  kubernetes/kubernetes:
+{%- set testgrid_dashboards = testgrid_dashboards + ", sig-node-presubmits" %}
+{%- endif %}
+{%- endif %}
+{%- if "crio" in job_name %}
+{%- set testgrid_dashboards = testgrid_dashboards + ", sig-node-cri-o" %}
+{%- endif %}
+{%- if "containerd" in job_name %}
+{%- set testgrid_dashboards = testgrid_dashboards + ", sig-node-containerd" %}
+{%- endif %}
+  - name: {{job_name}}
```

Comment:
So indention is the same for both periodic and presubmits?

The test bot seems to be stuck, but I suspect that a YAML linter would complain about that.

Reply:
yep, fortunately the indentation is the same for presubmits and periodics:
```
presubmits:
  kubernetes/kubernetes:
  - name: pull-kubernetes-e2e-containerd-gce
```

```
periodics:
  # This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently beta)
  # on a kind cluster with containerd updated to a version with CDI support.
  - name: ci-kind-dra
```

Reply:
No, if the lists were indented in the canonical way, it would be:

```
periodics:
- name: ci-kind-dra
```

YAML doesn't care, but there are stylecheckers which might.

Reply:
Let's see what https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/test-infra/34010/pull-test-infra-verify-lint/1870231711418159104 says.

Reply:
Well, I took that snipped from [the existing yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L1-L3).
And CI doesn't complain about wrong indentation for this file.

Reply:
Right. It runs yamllint, but that doesn't care, so we are good.

Reply:
As a last resort we can reindent in gen.py if it's really needed. It will be a little bit ugly though.
I suspect/hope that periodic and presubmit configs have the same indentation level in purpose.

Reply:
From a YAML perspective, the nesting level is different.

I don't remember anymore where, but there are other jobs where the indention is different, which is very annoying when copy-pasting from presubmit to periodic or vice-versa. That made me think that it's enforced. It's not, so it indeed makes much more sense to use the same indention even if it's not "quite right" for periodics.